params/Elvo_bird.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 50

```

Comment:
This value needs to be between 0 and 5, can you change it please?

Reply:
Sure. Doing that now.